typescript/cli/src/commands/warp.ts

```diff
@@ -144,6 +151,45 @@ export const read: CommandModuleWithContext<{
         context.registry,
         symbol,
       );
+
+      // TODO: merge with XERC20TokenAdapter and WarpRouteReader
+      const xerc20Limits = await Promise.all(
```

Comment:
Not sure if this is still the case but my intention was to keep these command definition files simple and dumb, in case we ever wanted to swap out yargs for something else. So I'd recommend moving this business logic to somewhere else